project/src/main/webapp/user-home.jsp

```diff
@@ -29,7 +37,9 @@
         <docs-component></docs-component>
       </div>
     </div>
-
+    <div class="sign-out">
+      <a href="/Auth"><button class="primary-blue-btn"> Sign out </button></a>
```

Comment:
Based on the way the Auth servlet works, you're better off making an AJAX call to it and outputting the result here instead of just linking to it. If you take a look at it, you'll see that it outputs a button with a link to singin/out which is why when you do this it takes you to a new page with just a button.

Reply:
Right sounds good!